src/envoy/auth/config.cc

```diff
@@ -122,25 +164,27 @@ IssuerInfo::IssuerInfo(Json::Object *json) {
   // Check "value"
   std::string value = json_pubkey->getString("value", "");
   if (value != "") {
+    pkey_ = std::unique_ptr<Pubkey>(new Pubkey());
     // Public key is written in this JSON.
     if (pkey_type_ == "pem") {
-      pkey_ = Pubkeys::CreateFromPem(value);
+      pkey_->Update(Pubkeys::CreateFromPem(value).release());
```

Comment:
You should be able to pass `unique_ptr` directly and use `std::move` inside.

Reply:
done